src/SFA.DAS.PR.Data.UnitTests/Repositories/AccountReadRepositoryTests.cs

```diff
@@ -42,4 +43,28 @@ public async Task AccountReadRepositoryTests_GetAccount_Valid()
 
         Assert.That(result, Is.Not.Null);
     }
+
+    [Test]
+    [InlineAutoData(123456, 123456, true)]
+    [InlineAutoData(654321, 1234567, false)]
+    public async Task AccountIdExists_Returns_Expected(long accountIdToCheck, long accountIdExisting, bool expected)
```

Comment:
Better way of doing this would be to create two constants one for valid id and one for invalid id and setup data context with valid id. This way you do not need to pass in two ids and the test becomes more readable. 
```
[InlineAutoData(validAccountId, true)]
    [InlineAutoData(invalidAccountId, false)]
```